core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/none.kt

```diff
@@ -52,3 +54,25 @@ public interface NoneColumnsSelectionDsl {
 }
 
 // endregion
+
+// region DataFrame
```

Comment:
For consistency with similar files, please put this function above the interface :)